Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Taxonomy of Failsafe Levels
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+
+## Abstract
+
+When talking about redundancy and backups in the context of cloud infrastructures, the scope under which circumstances these concepts apply to various ressources is neither homogenous nor intuitive.
+This decision record aims to define different levels of failure-safety.
+These levels can then be used in standards to clearly set the scope that certain procedures in e.g. OpenStack offer.
+
+## Terminology
+
+Image
+  OpenStack resource, virtual machine images usually residing in a network storage backend.
+Volume
+  OpenStack resource, virtual drive which usually resides in a network storage backend.
+Virtual Machine (abbr. VM)
+  IaaS resource, also called server, executes workloads from users.
```

Comment:
Why do we distinguish between IaaS resources and OpenStack resources? It there any added-value to categorize Virtual Machine as "IaaS Resource"